libr/debug/plugin.c

```diff
@@ -90,18 +107,42 @@ R_API bool r_debug_plugin_list(RDebug *dbg, int mode) {
 	return true;
 }
 
-R_API bool r_debug_plugin_add(RDebug *dbg, RDebugPlugin *foo) {
-	if (!dbg || !foo || !foo->meta.name) {
+R_API bool r_debug_plugin_add(RDebug *dbg, RDebugPlugin *plugin) {
+	if (!dbg || !plugin || !plugin->meta.name) {
```

Comment:
better r_return_val_if_fail (dbg && plugin); and make the runtime if for the meta.name

Reply:
good point!